build/init-topics-and-clients.sh

```
@@ -73,3 +83,9 @@ if [ ! -L build-clients -o "$(readlink build-clients)" != "$2" ]; then
 fi
 echo "Symlink to clients has been created at ./build-clients "
 
+#create symlink to AI libraries, except if it already exists
+if [ ! -L build-ai -o "$(readlink build-ai)" != "$3" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)